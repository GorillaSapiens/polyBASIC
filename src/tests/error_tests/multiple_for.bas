10 FOR i := 1 TO 10
20 NEXT i
30 FOR i := 1 TO 10
40 NEXT i
