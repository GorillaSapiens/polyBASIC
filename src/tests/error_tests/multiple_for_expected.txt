ERROR: @3:4, FOR VARIABLE MULTIPLY DEFINED ❮i❯ @1:4
